# It's a on_off srv
bool switch1
---
bool isDone

